recipes/getting-started/getting_started-image_semantic_segmentation.ipynb

```diff
@@ -0,0 +1,824 @@
+   "metadata": {},
+   "source": [
+    "<details>\n",
+    "<summary style=\"display: list-item;\"> Follow these instructions to import data from the API </summary>\n",
+    "\n",
+    "You should simply call [this function](https://cloud.kili-technology.com/docs/python-graphql-api/python-api/#append_many_to_dataset) :\n",
+    "\n",
+    "```python\n",
+    "kili.append_many_to_dataset(\n",
+    "    project_id=\"ckkpj7stx1bxc0jvk1gn9cu5v\", \n",
+    "    content_array=[\"path-to-local-image OR url-to-image\"],\n",
+    "    external_id_array=[\"your-identifier-of-the-image\"]\n",
+    ")\n",
+    "```\n",
+    "\n",
+    "\n",
+    "```python\n",
+    "# Example\n",
+    "\n",
+    "project_id = result['id']\n",
+    "\n",
+    "kili.append_many_to_dataset(\n",
+    "    project_id=project_id, \n",
+    "    content_array=[\"https://raw.githubusercontent.com/kili-technology/kili-playground/master/recipes/img/car_2.jpg\"],\n",
+    "    external_id_array=[\"car_2.jpg\"]\n",
+    ")\n",
+    "```\n",
+    "    \n",
+    "```python\n",
+    "Out: {'id': 'ckm4pmqmk0000d49k6ewu2um5'}\n",
+    "```\n",
+    "</details>"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Labeling <a id='labeling'></a>"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "When you created the project, you automatically became an admin of the project. This means that you can directly label. If you want to add members to the project, follow https://cloud.kili-technology.com/docs/projects/settings/#manage-project-members."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Label a specific asset"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "To annotate a specific asset, you can go to the dataset tab (in the side panel) :\n",
+    "\n",
+    "<img src=\"../img/sidebar_dataset.png\" width=100>\n",
+    "\n",
+    "![](../img/project_dataset.png)\n",
+    "\n",
+    "On the table of the assets, simply click on the line/asset (i.e., image here) you want to annotate"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Label the first asset in the queue"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Else, you can start to annotate right away with the `Start Labeling` button. "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### How to label ?"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Suppose that you've created a simple project as described in [Creating the project](#project). Now, to annotate, go to `Overview` and click on `Start Labeling`. You should arrive at the image of a car. Now just select a class (*Object A* or *Object B*) then you can drag the pointer around the object or make many straight lines until you finish to select an object.\n",
```

Comment:
Use correct names, not object A and Object B

Reply:
I changed the class for Door